### ISSUE TITLE
Not a bug report. It's a message to KeyMagic team.

### ISSUE THREAD

**Issue by kcvinker** [thantthet/keymagic#135]
Hi all,
I am a Keymagic user and it's working very well in my Windows 10 x64. But it's not working on my Windows 11 x64 laptop. So I filed an issue but it seems like there is no reply. So I have started studying to make a program like Keymagic. I am a fan of D programming language and after a 2-3 days googling and some experiments, I have succeeded in making a prototype in D. Currently it's working on Windows 11 x64 and Windows 10 x64.  This is to inform you that Keymagic will work on Windows11 if you find what's the problem., but you need to check what's happening exactly.

**Comment by SZTSIIT** [comment]
@kcvinker Hello, I know the post is old. But are you still having issues on Windows 11? Keymagic is working fine, though.

**Comment by kcvinker** [comment]
@SZTSIIT, Let me check.

**Comment by kcvinker** [comment]
@SZTSIIT  It's working when I changed some keyboard related settings in windows 11.

**Comment by SZTSIIT** [comment]
@kcvinker Thanks for your response back. I notice the zawcode keyboard doesn't have all Burmese consonants. And the Shift + Tide Key + Keys not working at the moment. I think Keyman fully supports most of the language.

**Comment by kcvinker** [comment]
@SZTSIIT, Yes, there are some minor probloms with hot keys. But that's trivial and easy to fix.  I wasted a lot of time for creating an alternative to KeyMagic.  See this checkbox in this image - 
![image](https://user-images.githubusercontent.com/8840907/217952438-527d9a20-607d-41e3-a90e-2a96b8a810ca.png)
This was unchecked and I didn't knew about this. After buying this laptop, when I realized that Keymagic is not working in Win11, I started making an alternative to it in the D programming language. After 3 weeks, I ended up a partial alternative. Because, I didn't build my code as dll, instead, I build it as an exe and that decision become an annoyance. Because, there is little program window which is hidden. And there is no way to indicate which keyboard is active now. But I used to nearly six months.
